src/components/primitives/Text/Text.js

```diff
@@ -1,38 +1,47 @@
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { space, variant } from 'styled-system';
+import { color, typography, space, variant } from 'styled-system';
+import { css } from '@styled-system/css';
 
 /**
  * A component to render all text in the app.
  */
 const Text = styled.p(
-	{
-		margin: 0,
-		listStyleType: 'none',
-		textAlign: props => props.align,
-	},
+	// Default styles
+	// css function provides access to our design tokens
+	css({
+		m: 0,
+		fontWeight: 'normal',
+		fontFamily: 'body',
+		fontSize: 'md',
+		lineHeight: 'base',
+		color: 'base',
+	}),
+	// APIs
+	color,
+	typography,
 	space,
 	variant({
 		variants: {
 			base: {
+				fontSize: 'md', // You can use design tokens in variants!
 				fontWeight: 'normal',
```

Comment:
Shouldn't we remove these lines now, since we have all shared props in `css()`?

Reply:
Yeah, I think it's simpler without it and just by using defaults without props.

Reply:
Yeah, I agree with that. Though importing `css` from another package isn't that nice.

Reply:
> importing css from another package isn't that nice.

Not sure what's the solution to that... any propositions?

Reply:
Wait until styled-system will export it or send them a PR? ;-) I think it should be in dependencies already since they added support of design tokens to variants.

Reply:
It won't be part of `styled-system` since it's what Theme-UI is doing. It is not in deps since we never used it before.